server/tserver/src/test/java/org/apache/accumulo/tserver/memory/LargestFirstMemoryManagerTest.java

```diff
@@ -204,22 +212,35 @@ protected long getMinCIdleThreshold(KeyExtent extent) {
     protected boolean tableExists(TableId tableId) {
       return true;
     }
+
+    @Override
+    protected boolean tableBeingDeleted(TableId tableId) {
+      return false;
+    }
   }
 
   private static class LargestFirstMemoryManagerWithExistenceCheck
       extends LargestFirstMemoryManagerUnderTest {
 
     Function<TableId,Boolean> existenceCheck;
+    Function<TableId,Boolean> deletingCheck;
 
-    public LargestFirstMemoryManagerWithExistenceCheck(Function<TableId,Boolean> existenceCheck) {
+    public LargestFirstMemoryManagerWithExistenceCheck(Function<TableId,Boolean> existenceCheck,
+        Function<TableId,Boolean> deletingCheck) {
       super();
       this.existenceCheck = existenceCheck;
+      this.deletingCheck = deletingCheck;
```

Comment:
It'd be better to use the primitive functional interface that maps to a boolean: `Predicate<TableId>` instead of `Function<TableId,Boolean>`.